WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/filter/ActivityLogTypeFilterFragment.kt

```diff
@@ -129,10 +129,11 @@ class ActivityLogTypeFilterFragment : DialogFragment() {
     }
 
     private fun refreshErrorScreen(uiState: Error) {
-        uiHelpers.setTextOrHide(actionable_empty_view.title, uiState.errorTitle)
-        uiHelpers.setTextOrHide(actionable_empty_view.subtitle, uiState.errorSubtitle)
-        uiHelpers.setTextOrHide(actionable_empty_view.button, uiState.errorButtonText)
-        actionable_empty_view.button.setOnClickListener { uiState.retryAction.action.invoke() }
+        actionable_empty_view.image.setImageResource(uiState.image)
```

Comment:
I didn't use ImageManager here since it was resizing the image to fullscreen and I didn't want to spent time on it. Any tips are welcome.

Reply:
I will be facing a similar issue with using image manager, but haven't given it any thought yet - probably next week.